packages/components/src/Toast/Toast.stories.tsx

```diff
@@ -13,6 +13,7 @@ const Template: Story<Args> = (args: Args) => <Toast {...args} />;
 
 const defaultArgs = {
   children: "You've got toast!",
+  onDismiss: () => console.log("dismissed!"),
```

Comment:
Should we leave this off and let Storybook supply it's own handler that'll work with the actions addon?

That's a little indirect/hidden, but it can also be kind of nice to tie in to Storybook.

Reply:
oh! yep, I'll try it again. honestly would be fine getting rid of the `NotDismissable` story anyways